README.md

```diff
@@ -1,11 +1,181 @@
+
+![A 1](https://user-images.githubusercontent.com/8396084/113326153-98aef380-9319-11eb-82da-ee930ee94c6a.jpg)
+
+
+We have determined the *Share of the CPU in the energy consumption of the Physical Machine* out of David Guyon's work. The distribution of the energy consumption page 33 highlights that 43% of the energy consumed by a physical machine is linked to the CPU. David Guyon cites *D. Kliazovich, P. Bouvry, and S. U. Khan, “GreenCloud: A Packet-level Simulator of Energyaware Cloud Computing Data Centers,” The Journal of Supercomputing, vol. 62, no. 3, pp. 1263–1283, 2012* which was published in 2012. This scientific base is quite old yet:
+- As David Guyon noted, this is the most recent paper he found on this subject in the literature;
+- We can certainely assume that the consumption distribution inside a physical machine hasn't evolved much over the past years.
+
+Then the rest of the calculation stands as following:
+
+![A 1 2](https://user-images.githubusercontent.com/8396084/113698547-decad500-96d4-11eb-8e7e-ba921a5c0fe7.jpg)
+
+
+
+With:
+- *i* beeing the type of instance considered (c5, m5, r5, ...)
+- *g* beeing the geographies where the observed cloud infrastructure has at least one instance running 
+- *v(i,g)* beeing the total volume of hours run by *i* instances in the *g* geography
+- *p(i)* beeing the power required by the instance's CPU at a 40-50% server utilization rate
+- *c(g)* beeing the carbon intensity of the electricity in the geography *g* where the considered data center is located
+
+
+#### A.2 Running compute primitives (EC2) - The _NRDC's_ approach
+
+We have named this second approach after the report published by the Natural Resources Defense Council (NRDC) in 2014 called Data Center Efficiency Assessment. A part of their study is focused on the level of utilization of IT equipment in data centers and we could find the following information: *for hyper-scale data centers, the server power at average utilization level stands at 101 watts*.
+
+Then, we calculate A.2 as below:
+
+![A 2](https://user-images.githubusercontent.com/8396084/113691744-12096600-96cd-11eb-8016-70f6f26b91cd.jpg)
+
+With:
+- *i* beeing the type of instance considered (c5, m5, r5, ...)
+- *g* beeing the geographies where the observed cloud infrastructure have at least one instance running 
+- *v(i,g)* beeing the total volume of hours run by *i* instances in the *g* geography
+- *r(i)* beeing the instance ratio of the instance considered (for example, a r5 instance has an *instance ratio* of 1/96)
+- *c(g)* beeing the carbon intensity of the electricity in the geography *g* where the considered data center is located
+
+
+
+### B. Running storage primitives (EBS, S3, ...)
```

Comment:
Some general remarks on the storage primitives:

- We should look into information about what kind of storage AWS uses for the different services, though it may be difficult. I think we can get from usage details for services like EC2 or EBS the type of storage (between SSD and HDD). It may be interesting to look at S3 and Glacier if some information is available.
- We should also look what redundancy is provided by the different services. For example on S3 objects may be replicated on several locations by default, so maybe it should be accounted N times where N is the number of copies.

Reply:
Maybe I should let this comment opened so we have this on mind for the next version?